types/codec.capnp

```diff
@@ -0,0 +1,32 @@
+@0xb02b1ca217480de7;
+
+ using Go = import "/go.capnp";
```

Comment:
This is a very nice syntax, but does this break compilation in other languages?
Will we need to add these little language-specific headers for each language we want to support?
Or copy/modify the `.capnp` files for each repo.

This is not *horrible* but does make it a bit harder for a script to ensure the codecs are identical